haystack/components/generators/azure.py

```diff
@@ -19,18 +19,21 @@
 @component
 class AzureOpenAIGenerator(OpenAIGenerator):
     """
-    A Generator component that uses OpenAI's large language models (LLMs) on Azure to generate text.
+    Generates text using OpenAI's large language models (LLMs).
 
-    It supports gpt-4 and gpt-3.5-turbo family of models.
+    It works with the gpt-4 and gpt-3.5-turbo models.
```

Comment:
```suggestion
    It works with the gpt-4 and gpt-3.5-turbo -type models.
```

Not sure if this is the correct punctuation in this case, but I'd mention that not only strictly these models can be used, but also similar GPT models.
I think I overlooked that in the other PRs.. 😬